### PR TITLE
Week4

### PR DESCRIPTION
**Pull request by FegorEO**

### CODE REVIEW

File: Econ490_PS1.do [hunk 1]
```diff
@@ -0,0 +1,141 @@
+*Week 4 Assignment
+*Oghenefegor Omorojor
+
+cd "/Users/user/Box Sync/Week4/04_assignment/data"
```
Comment (Benjamin-Tu) [inline Econ490_PS1.do:4]: might consider delete this line as it conflicts with global

File: Econ490_PS1.do [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+* Identifying village names
+gen household1 if mode_pixel = 1
+
+
+*Question 1c
+gen household2 if mode_pixel != 1 | pixel_consistent = 0
+gen household3 if mode_pixel != 1 | pixel_consistent = 1
+
+
+*Question 2
```
Comment (Benjamin-Tu) [inline Econ490_PS1.do:40]: The question 2 code is not working. You might consider using the psle_student_raw.dta directly and removing the data extraction part. Additionally, you need to create a loop that parses each school's information individually.

File: Econ490_PS1.do [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+	
+}
+
+*Identify school code
+global wd  "/Users/user/Box Sync"
+
+use "$wd/Week4/04_assignment/data/psle_student_raw", clear
+
+
+*Question 3
```
Comment (Benjamin-Tu) [inline Econ490_PS1.do:114]: Could you review question 3 again? The code is not working properly.

**Review by Benjamin-Tu — CHANGES_REQUESTED** [review]

File: Econ490_PS1.do (hunk 3 above)
Comment (FegorEO) [reply Econ490_PS1.do:114]: A redo of the assignment has been committed.

File: Econ490_PS1.do (hunk 2 above)
Comment (FegorEO) [reply Econ490_PS1.do:40]: A redo has been committed

File: Econ490_PS1.do (hunk 1 above)
Comment (FegorEO) [reply Econ490_PS1.do:4]: A redo has been committed

**Review by Benjamin-Tu — APPROVED** [review]
Thank you for making the changes. It would be great if you can add some comments to question 4 codes.

**Pull request closed without merging**